snaprecovery.sh

```diff
@@ -14,15 +14,24 @@ usage(){
 cat <<EOF
 usage: snaprecovery [-n, --no-merge] [SERIAL]
 
-The serial number of the device can be found by running 'adb devices'
+The serial number of the device can be found by running 'adb devices'.
+it is not necessary if only one device is connected in adb devices.
 
 Options:
     -n, --no-merge    don't merge videos with their respective overlays
 EOF
     exit 1
 }
 
-[ $# -eq 0 ] || [ "$1" = "" ] && usage
+# int of the number of device connected to adb
+DEVICESCOUNT="$(adb devices | awk 'NF && NR>1' | wc -l)"
+
+# if only one device is connected, use it's serial. otherwise the user is required to specify a serial.
```

Comment:
Fix the capitalizations and punctuations:
```
# If only one device is connected, use its serial, otherwise the user is required to specify a serial.
```